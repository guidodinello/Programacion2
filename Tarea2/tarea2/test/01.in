# Implementar structs, crearCadena, liberarCadena, inicioCadena
# Estas funciones siempre son llamadas por principal.cpp
Fin
