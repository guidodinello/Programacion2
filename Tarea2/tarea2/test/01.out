1>#  Implementar structs, crearCadena, liberarCadena, inicioCadena.
2>#  Estas funciones siempre son llamadas por principal.cpp.
3>Fin.
